ServiceBus/ServiceBus/ProductService.svc.cs

```diff
@@ -136,63 +141,74 @@ public Result AddProduct(string name, double price, Guid guid)
         /// <param name="guid">ID of a product</param>
         /// <param name="name">New name for a product</param>
         /// <param name="price">New price for a product</param>
+        /// <param name="pType">Type of a product</param>
         /// <returns>Modified product</returns>
-        public SharedLibs.DataContracts.Product EditProduct(Guid guid, string name, double price)
+        public SharedLibs.DataContracts.Product EditProduct(Guid guid, string name, double price, ProductType pType = null, bool enabled = true, bool headliner = false)
         {
             try
             {
-                var originalProduct = GetProduct(guid);

```

Comment:
this method should be much simpler. It should:
1) get the product using the DB context
2) change the values in retrieved product (instance from the db context)
3) return new instance of data contract "ServiceBus.DataContracts.Product" representing the changed item